multiagent/__init__.py

```diff
@@ -16,3 +18,12 @@
     entry_point='multiagent.envs:SimpleSpeakerListenerEnv',
     max_episode_steps=100,
 )
+
+warnings.warn("This code base is no longer maintained, and is not expected to be maintained again in the future. \n"
+              "For the past handful of years, these environments been maintained inside of PettingZoo (see "
+              "https://pettingzoo.farama.org/environments/mpe/). \nThis maintained version includes documentation, "
+              "support for the PettingZoo API, support for current versions of Python, numerous bug fixes, \n"
+              "support for installation via pip, and numerous other large quality of life improvements. \nWe "
+              "encourage researchers to switch to this maintained version for all purposes other than comparing "
+              "to results run on this version of the environments. \n")
+input("Please read the raised warning, then press Enter to continue...\n")
```

Comment:
this would trigger (request user to press Enter) every single time you run the code though, right? I agree with everything except for this

Reply:
> this would trigger (request user to press Enter) every single time you run the code though, right? I agree with everything except for this

The issue that led to this was that other writes to stdout drowns out this warning. Alternatively, we could use `sleep()`.

What are your thoughts?

Reply:
Could also use environment variables instead/as well as pressing Enter.

Reply:
ok I can live with that!